src/main/java/com/iexec/standalone/chain/Web3jService.java

```diff
@@ -33,4 +35,18 @@ public Web3jService(ChainConfig chainConfig) {
         );
     }
 
+    public Web3jService(BlockchainAdapterConfigurationService blockchainAdapterConfigurationService,
```

Comment:
This constructor should not be necessary, the Spring bean would already be created with the previous constructor